km62/Antonov_Mykhailo/laboratory1.sql

```diff
@@ -1,2 +1,126 @@
 -- LABORATORY WORK 1
 -- BY Antonov_Mykhailo
+
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+модифікувати таблиці та вставляти дані
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+
+CREATE USER antonov IDENTIFIED by antonov
+DEFAULT TABLESPACE "SYSTEM"
+TEMPORARY TABLESPACE "TEMP";
+
+GRANT "CONNECT" TO antonov ;
+GRANT "DATAPUMP_IMP_FULL_DATABASE" TO antonov ;
+GRANT "IMP_FULL_DATABASE" TO antonov ;
+
+
+
+
+
+
+
+
+
+
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Студент купляє квиток на потяг.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+
+CREATE TABLE TABLE1 
+(
+  NAME_STUDENT VARCHAR2(20) NOT NULL,
+  TICKET_NUMBER VARCHAR2(20) 
+);
+
+
+ALTER TABLE TABLE1 
+ADD CONSTRAINT PRIMARY KEY TICKET_NUMBER;
+
+
+  
+/* --------------------------------------------------------------------------- 
+3. Надати додаткові права користувачеві (створеному у пункті № 1) для створення таблиць, 
+внесення даних у таблиці та виконання вибірок використовуючи команду ALTER/GRANT. 
+Згенерувати базу даних використовуючи код з теки OracleScript та виконати запити: 
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+
+GRANT ALTER ANY TABLE TO  antonov;
+GRANT UPDATE ANY TABLE TO  antonov;
```

Comment:
Read the task again. You need to use 'CREATE' and 'SELECT' with GRANT. If you have done task number one (creating user scheme) - you don`t need to write 'INSERT'. Otherwise, do it.